Nodejs/Product/Nodejs/SharedProject/OutputWindowWrapper.cs

```diff
@@ -24,41 +24,37 @@ public sealed class OutputPaneWrapper
         private SVsServiceProvider ServiceProvider { get; set; }
 
         private Dictionary<OutputWindowTarget, IVsOutputWindowPane> lazyOutputPaneCollection = new Dictionary<OutputWindowTarget, IVsOutputWindowPane>();
-        private IVsWindowFrame lazyOutputWindow;
 
-        private IVsOutputWindowPane GetOutputPane(OutputWindowTarget target)
+        public OutputPaneWrapper()
         {
-            if (this.lazyOutputPaneCollection.TryGetValue(target, out var lazyOutputPane))
-            {
-                return lazyOutputPane;
-            }
-            else
-            {
-                ThreadHelper.ThrowIfNotOnUIThread();
-                var (title, guid) = GetPaneInfo(target);
-                var outputWindow = (IVsOutputWindow)this.ServiceProvider.GetService(typeof(SVsOutputWindow));
+            //...
+        }
 
-                // Try to get the workspace pane if it has already been registered
-                var hr = outputWindow.GetPane(guid, out lazyOutputPane);
+        private void InitializeOutputPane(OutputWindowTarget target)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+            var (title, guid) = GetPaneInfo(target);
+            var outputWindow = (IVsOutputWindow)this.ServiceProvider.GetService(typeof(SVsOutputWindow));
 
-                // If the workspace pane has not been registered before, create it
-                if (lazyOutputPane == null || ErrorHandler.Failed(hr))
-                {
+            // Try to get the workspace pane if it has already been registered
+            var hr = outputWindow.GetPane(guid, out var lazyOutputPane);
 
-                    if (ErrorHandler.Failed(outputWindow.CreatePane(guid, title, fInitVisible: 1, fClearWithSolution: 1)) ||
-                        ErrorHandler.Failed(outputWindow.GetPane(guid, out lazyOutputPane)))
-                    {
-                        return null;
-                    }
+            // If the workspace pane has not been registered before, create it
+            if (lazyOutputPane == null || ErrorHandler.Failed(hr))
+            {
 
-                    // Must activate the workspace pane for it to show up in the output window
-                    lazyOutputPane.Activate();
+                if (ErrorHandler.Failed(outputWindow.CreatePane(guid, title, fInitVisible: 1, fClearWithSolution: 1)) ||
+                    ErrorHandler.Failed(outputWindow.GetPane(guid, out lazyOutputPane)))
+                {
+                    return;
                 }
 
-                this.lazyOutputPaneCollection.Add(target, lazyOutputPane);
-                return lazyOutputPane;
+                // Must activate the workspace pane for it to show up in the output window
+                lazyOutputPane.Activate();
             }
 
+            this.lazyOutputPaneCollection.Add(target, lazyOutputPane);
```

Comment:
>this.lazyOutputPaneCollection.Add(target, lazyOutputPane); [](start = 12, length = 58)

If this is the only place where this method mutates `this`, it might be cleaner to return the result and have the caller update the map.  In fact, rather than pass `target` in at all, why not inline the results of `GetPaneInfo` at the only two call sites?